src/main/java/org/usfirst/frc/team5687/robot/Constants.java

```diff
@@ -70,6 +75,18 @@
         public static final double BOWL_SPEED = -1.0;
     }
 
+    public class Shooter {
+        /**
+         * Time for running shooter wheels for priming
+         */

```

Comment:
5s is a long time to wait, is this really the required length of time for the shooter to spin up?